doc/sphinx/Pacemaker_Explained/rules.rst

```diff
@@ -931,20 +1125,8 @@ business activities.
          </meta_attributes>
       </rsc_defaults>
 
-Rules may be used similarly in ``instance_attributes`` or ``utilization``
-blocks.
-
-Any single block may directly contain only a single rule, but that rule may
-itself contain any number of rules.
-
-``rsc_expression`` and ``op_expression`` blocks may additionally be used to
-set defaults on either a single resource or across an entire class of resources
-with a single rule. ``rsc_expression`` may be used to select resource agents
-within both ``rsc_defaults`` and ``op_defaults``, while ``op_expression`` may
-only be used within ``op_defaults``. If multiple rules succeed for a given
-resource agent, the last one specified will be the one that takes effect. As
```

Comment:
> If multiple rules succeed for a given resource agent, the last one specified will be the one that takes effect.

If that is actually true, we should keep it documented somewhere. I don't recall seeing it.

Reply:
It isn't